### ISSUE TITLE
color contrast for success status 

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#2168]
Currently, the [success version of a progress bar](https://www.patternfly.org/v4/documentation/react/components/progress/#progress-success) includes a success icon that uses the color `--pf-global--success-color--100` on a white background. This color fails color contrast, and should be replaced with a color that passes the color contrast check.

**Comment by serenamarie125** [comment]
@mceledonia @mcarrano @jeff-phillips-18 FYI

**Comment by serenamarie125** [comment]
@jgiardino this is not constrained to the progress bar, it's also an issue with some of the alerts correct?

**Comment by jgiardino** [comment]
Correct. The [inline alerts](https://www.patternfly.org/v4/documentation/core/components/alert#inline-alert-types) for success don't provide enough color contrast for the icon on white.

**Comment by mceledonia** [comment]
I believe I have a solution for this outlined in my comment from 10/9/2019 at  #2320 @jgiardino @serenamarie125

**Comment by mcarrano** [comment]
I believe this was resolved by #2320. Closing.
